docs/pages/api-reference/operators/window.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Window
+order: 0
+status: published
+---
+### Window
+
+<Divider>
+<LeftSection>
+Operator to do aggregation of events into windows based on their timestamps. Window operator must always be preceded by a [groupby](/api-reference/operators#groupby) operator.
+
+#### Parameters
+
+<Expandable title="type" type="Window">
+
+The type of window to group event on. Possible values are `"hopping"`, `"tumbling"`, `"session"`.
+
+`"hopping"`: Take in two parameters `"duration"` and `"stride"`, the `"duration"` parameters indicate how long the window is while `"gap"` indicate how far apart the window should be. Hopping window can overlap meaning one event can belong to two different windows. `"stride"` should be smaller than `"duration"`.
+
+`"tumbling"`: Take in one parameter `"duration"` indicate how long the window is. The tumbling windows do not overlapped and cover the whole range. 
```

Comment:
Tumbling windows are characterized by a single parameter: duration.

duration: Specifies how long each window lasts. Unlike hopping windows, tumbling windows are designed to be non-overlapping and contiguous, ensuring that each event is placed into exactly one window, covering the entire data range without gaps.